catalogs/catalog-hive/src/main/java/org/apache/gravitino/catalog/hive/HiveTable.java

```diff
@@ -143,7 +143,8 @@ private static Map<String, String> buildTableProperties(Table table) {
   /**
    * Converts this HiveTable to its corresponding Table in the Hive Metastore.
    *
-   * @return The converted Table.
+   * @param tablePropertiesMetadata metadata for configuring the Hive Metastore table
+   * @return the converted Table
```

Comment:
Please leave any sentances as is. While the Javadocs are inconsistent, they should use sentences.